README.md

```diff
@@ -1,2 +1,253 @@
-# BudgetPro
-BudgetPro is a mobile app that list your money when ever you paid for some things.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>BudgetPro</b></h3>
+</div>
+
+[![codecov](https://codecov.io/gh/cancelei/blog-in-rails/graph/badge.svg?token=ITBDQ9ZRO7)](https://codecov.io/gh/cancelei/blog-in-rails)
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 BudgetPro ](#ruby-on-rails-group-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BudgetPro<a name="ruby-on-rails-group-project"></a>
+
+**BudgetPro** 
+The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+You can see an introduction of this webpage in this [video](https://drive.google.com/file/d/1oBzqc8N57oS7Pg359pGpcTlULL4Htnqs/view?usp=sharing)

```

Comment:
- You did well making a video and taking users through the features of your project, you also did well showing what you are proud of implementing, but according to the project requirements, you are to show the `piece of code` you were proud of implementing not the functionality you are proud of implementing. I suggest you do the video again and show the code you are proud to implement. Please fix this 👍 👍